lib/node.js

```diff
@@ -136,7 +136,10 @@ Spinal.prototype.start = function(cb) {
           })
         }
         if (err) {
-          res.status(500).send({ message: err.message, stack: err.stack, data: respData, header: header })
+          var returnedError = { message: err.message, stack: err.stack, data: respData, header: header }
+          if (typeof err.type !== 'undefined') returnedError.type = err.type
+          if (typeof err.source !== 'undefined') returnedError.source = err.source
```

Comment:
Is these 2 lines `null` safe?

Reply:
I think it's up to the sender (can be null).